src/ui/pages/BackupWallet/BackupSettingsItem/BackupSettingsItem.tsx

```diff
@@ -75,6 +76,7 @@ export function BackupFlowSettingsSection() {
                       to={`/backup-wallet?groupId=${group.id}&backupKind=verify`}
                       size={32}
                       style={{ paddingLeft: 16, paddingRight: 16 }}
+                      onClick={openInTabView}
```

Comment:
Right now this method doesn't work well in Arc :(
It opens new empty tab that required manual reload.
It can be fixed the same way as `openInFullScreen` works

Reply:
Thanks!